programs/token-metadata/program/src/state/mod.rs

```diff
@@ -200,7 +200,7 @@ pub trait Resizable: TokenMetadataAccount + BorshSerialize {
         // passes a slice to borsh so the internal account data array does not get
         // temporarily resized
         let mut storage = &mut account_data[..required_size];
-        BorshSerialize::serialize(self, &mut storage)?;
+        borsh::to_writer(&mut storage, self)?;
```

Comment:
So should this also just be reduced to?:
```
borsh::to_writer(&mut account_info.data.borrow_mut()[..], self)?;
```
And remove the comment that is now stale?

Reply:
Good point, it has been fixed now.